src/main/kotlin/io/kotest/plugin/intellij/KotestConfiguration.kt

```diff
@@ -179,6 +188,8 @@ class KotestConfiguration(name: String, factory: ConfigurationFactory, project:
 
    override fun readExternal(element: Element) {
       super.readExternal(element)
+      alternativeJrePath = JDOMExternalizerUtil.readField(element, AlternativeJrePathField)
```

Comment:
Are we sure this will work if the existing value is null?
If so, then all good to merge.

Reply:
I'm not really sure how to set it to null. Closest I got was selecting some empty line in the JRE selector, saving the configuration and then loading it again and it would load back as null without issue.